release.toml

```diff
@@ -6,7 +6,8 @@ sign-tag = true
 publish = false
 
 pre-release-replacements = [
-  { file = "README.md", search = "saxaboom = .*", replace = "{{crate_name}} = \"{{version}}\"" },
+  { file = "README.md", search = "saxaboom = .*", replace = "saxaboom = \"{{version}}\"", min = 0 },
+  { file = "README.md", search = "saxaboom-runtime = .*", replace = "saxaboom-runtime = \"{{version}}\"", min = 0 },
```

Comment:
We have to tell `cargo-release` to allow a mismatch; we cannot have a per-crate replacement with `{{crate_name}}` :/

It's a pretty dumb tool too, I have to use `cargo release 0.1.0 --workspace --exclude bindings_generator` otherwise it tries to publish that `publish = false` crate too and fails :upside_down_face: